packages/report-server/examples.http

```diff
@@ -1,13 +1,62 @@
-@hostname = localhost
-@port = 8030
-@host = {{hostname}}:{{port}}
-@version = v1
+@baseUrl = http://localhost:8030/v1
+# Uncomment below to fetch against the dev server, rather than localhost
+# @baseUrl = https://dev-report-api.tupaia.org/v1
```

Comment:
I generally find that in reality, rather than changing the version or port etc. I am more likely to flick between dev and localhost, so this is a more practical variable to save (especially with the change from http to https)